eva-accession-import-dbsnp2/pom.xml

```diff
@@ -59,6 +52,16 @@
             <groupId>uk.ac.ebi.eva</groupId>
             <artifactId>variation-commons-batch</artifactId>
         </dependency>
```

Comment:
Please solve conflicts in this file due to other PRs being recently merged, otherwise Travis will not be triggered.